src/test/javascript/cypress/integration/account/password-page.spec.ts

```diff
@@ -17,8 +17,8 @@ describe('/account/password', () => {
     cy.visit('/');
     cy.login('user', 'user');
   });
-
-  beforeEach(() => {
+  //code commented because this view is not present and it will be changed in a future feature
```

Comment:
Maybe leave a `// TO DO: ` with the story id where this test should be fixed. Add this in all commented tests.